### PR TITLE
fix(input): make autosize work inside tabs & stepper

### LINKED ISSUE #7316: md-autosize initialized with height of 0 inside stepper & tabs
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
TextArea indicator should always show onload

#### What is the current behavior?
TextArea indicator with md-autosize not showing inside the Stepper.
The indicator will only show when users starts to type into it.

#### What are the steps to reproduce?
[Plunker Link](https://plnkr.co/edit/OwCi5QPmaWv8htG8gaWQ?p=preview)

Step 1: Notice that the TextArea outside the stepper is showing the indicator and the TextArea inside the Stepper is not showing the indiciator.
Step 2: Start to type into the TextArea inside the stepper

TextArea Indicator Error

#### What is the use-case or motivation for changing an existing behavior?
NA

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Material Beta 11

#### Is there anything else we should know?
NA

### PR DESCRIPTION
**Pull request by mmalerba**
fixes #7316

### CODE REVIEW

**Comment by jelbourn** [comment]
Actually, add a unit test for stepper + autosize?

**Comment by mmalerba** [comment]
I feel like this kind of thing is better caught with a screenshot test, which I did for tabs. Didn't add it to the stepper e2e test because it uses the stepper overview example, and I didn't want to start adding irrelevant stuff to the example

**Comment by jelbourn** [comment]
An easy sanity test would be to make the initial text really long and assert that the height is above some small-ish threshold

**Comment by mmalerba** [comment]
done

File: src/lib/input/autosize.spec.ts [hunk 1]
```diff
@@ -202,6 +213,20 @@ describe('MdTextareaAutosize', () => {
     expect(textarea.clientHeight)
         .toBeGreaterThan(previousHeight, 'Expected the textarea height to have increased.');
   }));
+
+  it('should work in a tab', () => {
+    const fixtureWithForms = TestBed.createComponent(AutosizeTextareaInATab);
+    fixtureWithForms.detectChanges();
+    textarea = fixtureWithForms.nativeElement.querySelector('textarea');
+    expect(textarea.getBoundingClientRect().height).toBeGreaterThan(0);
```
Comment (jelbourn) [inline src/lib/input/autosize.spec.ts]: Would greater than `1` work?

**Pull request merged**